Dockerfile

```diff
@@ -57,11 +54,10 @@ RUN useradd -U -m superset && \
         unzip \
         python-dev && \
     apt-get clean -y && \
-    rm -rf /var/lib/apt/lists/* /tmp/google-chrome-stable_current.deb /tmp/chromedriver.zip
+    rm -rf /var/lib/apt/lists/*
 
 COPY superset /usr/local/bin
 WORKDIR /home/superset
 
 EXPOSE 8088
-CMD ["superset-run"]
-USER superset
+CMD ["superset-run"]
```

Comment:
why remove the superset user?

Reply:
I was overlooking the severness of not using a non-privileged user.
just reminded myself again of it: https://www.docker.com/blog/understanding-the-docker-user-instruction/

added it again